src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -86,10 +84,63 @@ private static bool ShouldIncludeInTargetTypedCompletionList(
             return isConvertible;
         }
 
-        typeConvertibilityCache[type] = CompletionUtilities.IsTypeImplicitlyConvertible(semanticModel.Compilation, type, inferredTypes);
+        typeConvertibilityCache[type] = CompletionUtilities.IsTypeImplicitlyConvertible(syntaxContext.SemanticModel.Compilation, type, syntaxContext.InferredTypes);
         return typeConvertibilityCache[type];
     }
 
+    private static bool ShouldIncludeInTargetTypedCompletionListInNamedType(ISymbol symbol, TSyntaxContext syntaxContext, Dictionary<ITypeSymbol, bool> typeConvertibilityCache)
```

Comment:
the parent is only ever callint this with an INamedTypeSymbol, can you change sig to match?

Reply:
Nice catch!